esp8266/Makefile

```diff
@@ -51,6 +54,7 @@ SRC_C = \
 	modpybpin.c \
 	modesp.c \
 	utils.c \
+	$(BUILD)/frozen.c \
```

Comment:
This looks wrong: the makefile will prepend $(BUILD) again to this when making the list of .o files...